bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -93,57 +93,57 @@ To send push notifications, the notification actions have to be used in rules.
 
 Three different actions are available:
 
-- `sendNotification(emailAddress, message)`: Send a notification to a _specific_ openHAB Cloud user.
-- `sendBroadcastNotification(message)`: Send a broadcast notification to _all_ devices of _all_ users.
-- `sendLogNotification(message)`: Send a log notification to the notifications list. Log notifications do not trigger a notification on the device.
+* `sendNotification(emailAddress, message)`: Send a notification to a _specific_ openHAB Cloud user.
```

Comment:
Why in the previous file (homekit) you replaced * by - and here you do the reverse. The rendering seems to be identical, no ?

Reply:
It should be "- " rather than "* ", see for example #14371.

Reply:
I’ll adapt, as being repo wide consistent is a good thing. For the warnings, lifting or rendering it does not matter. 
I’ll push the changes tomorrow morning

Reply:
Yes, you are right that is doesn't have any rendering impact, I believe the previous documentation refactoring was for consistency.

Reply:
Changed